cpu/cc2538/vectors.c

```diff
@@ -71,74 +71,74 @@ WEAK_DEFAULT void isr_dmaerr(void);
 /* interrupt vector table */
 ISR_VECTORS const void *interrupt_vector[] = {
     /* Exception stack pointer */
-    (void*) (&_estack),             /* pointer to the top of the stack */
+    __extension__ (void*) (&_estack),             /* pointer to the top of the stack */
```

Comment:
> Isn't the root cause for the warning that ISO C forbids putting a function pointer in void*?

Yes.

> Silencing the warning like this seems closing eyes

Partly. Yes, it is not standard compliant, but in practice there's no real reason against it as far as I have understood the problem. The difficulty here is, that we're mixing function pointers and pointers to fixed memory positions here. Without the pointer to _estack etc. we could redefine the array.